src/common/Common.hpp

```diff
@@ -12,6 +12,9 @@
 #define LINK_CHATTERINO_DISCORD "https://discord.gg/7Y5AYhAK4z"
 #define LINK_CHATTERINO_SOURCE "https://github.com/Chatterino/chatterino2"
 
+const QString FSTRING_TWITCH_PLAYER_URL =
```

Comment:
Rename to `TWITCH_PLAYER_URL` and move into the chatterino namespace

Reply:
This should be declared `inline` and possibly use `QStringLiteral` to avoid allocating at runtime.